bin/ci_tools/markdown_linter.py

```diff
@@ -20,6 +20,16 @@ def index_to_line_info(idx, text_block):
     char = idx - max(text_block[:idx].rfind('\n'), 0)+1
     return LineInfo(line, char)
 
+def find_markdown_files(folder):
+    """
+    Find all.md files in a given folder and its subfolders.
+    """
+    md_files = []
+    for root, _, files in os.walk(folder):
+        for file in files:
+            if file.endswith('.md'):
+                md_files.append(os.path.join(root, file))
+    return md_files
```

Comment:
This is simpler with the `glob` package
```suggestion
    return list(glob.glob(f'{folder}/**/*.md'))
```